src/components/ImageUpload.js

```diff
@@ -54,7 +52,7 @@ export default function ImageUpload({ inputData }) {
           <CancelButton onClick={handleRemoveImage}>
             <img src={cancel} width="35" alt="Thumb" />
           </CancelButton>
-          <img src={inputData.img} alt="Delete" style={{ width: '100%', cursor: 'pointer' }} />
+          <img src={image} alt="Delete" style={{ width: '100%', cursor: 'pointer' }} />
```

Comment:
Why use inline styles here? I would also assign height and width here. And also styled-components